### DISCUSSION TITLE
good job

### DISCUSSION THREAD

**Discussion by klikis** [jekhokie/raspberry-noaa-v2#73] [Show and tell]
great job, it works great
![Cattura](https://user-images.githubusercontent.com/9518512/107778322-3e93a800-6d44-11eb-8758-99a68e985195.PNG)
![NOAA-18-20210212-100754-MCIR-precip](https://user-images.githubusercontent.com/9518512/107778395-50754b00-6d44-11eb-8668-03ff6a0edd99.jpg)
![METEOR-M-2-20210212-081923-122-rectified](https://user-images.githubusercontent.com/9518512/107778441-5ec36700-6d44-11eb-8c57-9555abe15fd1.jpg)

**Comment by jekhokie** [comment]
Thanks @klikis, glad it's working out for you!

**Reply by klikis** [reply]
thanks to you.

**Reply by ON6PH** [reply]
Nice! Is there something you did to get the Meteor images in color? I only receive IR images.
